docs/articles/guides/how-it-works.md

```diff
@@ -35,6 +36,7 @@ IEnumerable<Results> Run(Benchmark benchmark)
 Result ActualRun(Method method, Job job)
 {
     GlobalSetup();
+    // JittingStage(method) // triggers JIT compilation (and tiering if enabled) before Pilot/Warmup
```

Comment:
This is pseudo-code, so you can add the method to demonstrate how it behaves like the other methods do.